pkg/coderefs/coderefs.go

```diff
@@ -20,6 +20,8 @@ import (
 // PUTing a massive json payload. These limits will likely be tweaked over
 // time. The LaunchDarkly backend will also apply limits.
 const (
+	version = "0.6.0"
```

Comment:
i know we need this but its unfortunate that we have yet another thing we need to update every single time. Perhaps RELEASING.md should have a checklist of all lines of code that need updating, or a sed script that does it or something